packages/terra-demographics-banner/src/_LargeDemographicsBannerDisplay.jsx

```diff
@@ -46,31 +56,46 @@ export default (props) => {
   delete customProps.className;
 
   return (
-    <section className={mainClasses} {...customProps}>
+    <section
+      {...customProps}
+      className={mainClasses}
+      tabIndex={-1}
+    >
       {props.photo && (
         <div className={cx('profile-photo')}>
           {props.photo}
         </div>
       )}
       <div className={cx('content')}>
-        <div className={cx('row')}>
+        <div className={cx('column')}>
           <h1 className={cx('person-name')}>
+            { isConfidential ? <span className={cx('confidential-icon')}><IconFlag /></span> : null }
             { personName }
             { preferredFirstName && <span className={cx('preferred-first-name')}>{ preferredFirstName }</span> }
           </h1>
-          <div className={cx('application-content')}>
-            {applicationContent}
-          </div>
-        </div>
-        <div className={cx('row')}>
           <div className={cx('person-details')}>
             {DemographicsBannerUtils.personDetails(props)}
           </div>
+        </div>
+        <div className={cx('column')}>
+          <div className={cx('application-content')}>
+            {applicationContent}
+          </div>
           <div className={cx('identifiers')}>
             {DemographicsBannerUtils.applicationIdentifiers(props)}
           </div>
         </div>
       </div>
+      {isSelectable && (
+        <div
+          className={cx('info-tile-container')}
+          tabIndex={0} // eslint-disable-line jsx-a11y/no-noninteractive-tabindex
```

Comment:
This info-tile-container is showing focus a focus outlinie when it is explicitly clicked. Should we be doing the same 'should show focus' logic here that we're performing for the other buttons?

Reply:
The focus on the info tile now no longer show focus when explicitly clicked. It shows the focus only with keyboard interaction. https://github.com/cerner/terra-core/pull/2856/commits/27fb162fbd852f4ceb819ec25cfbc3eee36e06f0